less/index.less

```diff
@@ -1 +1,41 @@
 // Start here!
+
+// General styles
+.navbar-light {
+    background-color: rgb(0, 141, 146);
+}
+h1 {
+    text-align: center;
+}
+
+.jumbotron {
+    margin: 0 auto;
+    background-color: rgb(231, 233, 233);
+}
+
+.navbar {
```

Comment:
There's no need to nest this deep like this. Also you're repeating code that's on line 21 in line 35. You can simplify everything from line 16—41 to fewer lines of code:

```
.btn-group .nav-item .btn a {
    color: white;
}   

.btn {
    background: green;
    border: none;
    &:hover {
        background: green * 1.25;
    }
}
```